libmisc/find_new_gid.c

```diff
@@ -112,6 +113,11 @@ static int check_gid (const gid_t gid,
 		return ERANGE;
 	}
 
+  /* Check for compatibility with 16b and 32b gid_t error codes */
+	if (gid == (u_int16_t) -1 || gid == (u_int32_t) -1) {
```

Comment:
Where is `u_int16_t` defined? Maybe it's a typo for `uint16_t`?

Reply:
BTW, you could get those values as `UINT16_MAX` and `UINT32_MAX` and avoid any casts.

Reply:
To be honest I have no idea, probably just a completion typo or something. Looks like it is both in glibc headers and in linux, but It was meant to be `uintX_t`.